qiskit/ignis/verification/quantum_volume/fitters.py

```diff
@@ -302,32 +302,66 @@ def plot_qv_data(self, ax=None, show_plt=True):
             plt.show()
 
     def qv_success(self):
-        """Return whether each depth was successful (>2/3 with confidence
-        greater than 97.5) and the confidence
+        """Return whether each depth was successful (> 2/3 with confidence
+        level > 0.977 corresponding to z_value = 2) and the confidence level
 
         Returns:
             list: List of lenth depth with eact element a 3 list with
             - success True/False
-            - confidence
+            - confidence level
         """
 
         success_list = []
+        confidence_level_threshold = self.calc_confidence_level(z_value=2)
 
         for depth_ind, _ in enumerate(self._depths):
             success_list.append([False, 0.0])
             hmean = self._ydata[0][depth_ind]
-            if hmean > 2/3:
-                cfd = 0.5 * (1 +
-                             math.erf((hmean - 2/3)
-                                      / (1e-10 +
-                                         self._ydata[1][depth_ind])/2**0.5))
-                success_list[-1][1] = cfd
+            sigma = self._ydata[1][depth_ind]
```

Comment:
So the 1e-10 I had before prevents an error if sigma=0 (e.g. if there's only 1 trial). You could alternatively just run an if for that case.

Reply:
Thanks for your comment! I will add an if to handle error.